R/pkg/inst/tests/testthat/test_mllib_regression.R

```diff
@@ -379,6 +379,49 @@ test_that("glm save/load", {
   unlink(modelPath)
 })
 
+test_that("spark.glm and glm with string encoding", {
```

Comment:
we are trying to cut down to only a core set of tests to run on CRAN.
please add `skip_on_cran()` here